arch/arm/core/Kconfig

```diff
@@ -164,3 +167,7 @@ source "arch/arm/core/cortex_m/Kconfig"
 if CPU_HAS_MPU
 source "arch/arm/core/cortex_m/mpu/Kconfig"
 endif
+
+if CPU_HAS_TEE
+source "arch/arm/core/cortex_m/tz/Kconfig"
```

Comment:
Is this include (and CPU_HAS_TEE) needed? The tz/Kconfig doesn't seem to contain any relevant additional config options.

Reply:
It actually, contains config ARM_TRUSTZONE_M which is the way for the user to select TEE support in their ARM board.